packages/eas-cli/src/graphql/queries/ProjectQuery.ts

```diff
@@ -0,0 +1,28 @@
+import gql from 'graphql-tag';
+
+import { graphqlClient, withErrorHandlingAsync } from '../client';
+import { Project } from '../types/Project';
+
+type ProjectIdType = Pick<Project, 'id'>;
+
+export class ProjectQuery {
+  static async idByUsernameAndSlugAsync(username: string, slug: string): Promise<ProjectIdType> {
+    const data = await withErrorHandlingAsync(
+      graphqlClient
+        .query<{ project: { byUsernameAndSlug: ProjectIdType } }>(
+          gql`
+          {
+            project {
+              byUsernameAndSlug(username: "${username}", slug: "${slug}", sdkVersions: []) {
```

Comment:
Please use variables.

Reply:
```suggestion
              byUsernameAndSlug(username: $username, slug: $slug) {
```
You can also omit `sdkVersions` because it's an optional parameter.

Reply:
Indeed, this is a leftover after a bug in `expo/universe`, this parameter was required to avoid crashes, but now it can be omitted.

Regarding to variables, I was wondering why they weren't used before and finally I just sticked to existing string interpolation approach. But I'm glad you're in favour of using them.